### ISSUE TITLE
Rewrite redirect for 303 status code

### ISSUE THREAD

**Issue by itaied246** [nodejitsu/node-http-proxy#1129]
In the rewrite [redirect](https://github.com/nodejitsu/node-http-proxy/blob/master/lib/http-proxy/passes/web-outgoing.js) there are these status codes to redirect by:

    (line 5) var redirectRegex = /^201|30(1|2|7|8)$/;

I think you should also include the 303 (`SEE OTHER`) response status.
What do you think?

**Comment by themasch** [comment]
ran into the exact same problem.

**Comment by reflexator** [comment]
Yep there is pull request https://github.com/nodejitsu/node-http-proxy/pull/1125
